src/test/java/com/google/javascript/gents/multiTests/goog_provide_rewrite/export.ts

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
That's not easy to check, but in practice that won't actually happen because most files start with `import` which doesn't have a newline prepended.